draft-ietf-cbor-packed.md

```diff
@@ -262,12 +262,15 @@ affix data item (affix, which might need to be recursively unpacked
 first) "concatenated" with the tag content (rump, again possibly
 recursively unpacked).
 
-* For a rump of type array and map, the affix also needs to be an
-  array or a map.
-  For an array, the elements from the prefix are prepended to the rump
-  array, while the elements from a suffix are appended.
-  For a map, the entries in the affix are added to those of the rump;
-  prefix and suffix references differ in how entries with identical
+* For a rump of type array, the affix also needs to be an
+  array.
+  The elements from a prefix are prepended to the elements in the rump
+  array, while the elements from a suffix are appended to those in the
+  rump.
+
+* For a rump of type map, the affix also needs to be a map.
+  The entries in the affix are added to those of the rump.
```

Comment:
The CBOR-based protocol in question might be using some deterministic format (see Section 4.2 of RFC 8949). If that's the case, any deterministic restriction related to map keys (e.g., their encoding and sorting) also apply when adding these entries to the rump, right?

Reply:
That is a great question for the discussion in 80 minutes...